src/rendering/RenderUtil.cc

```diff
@@ -2504,7 +2504,27 @@ void RenderUtil::Init()
     return;
 
   gz::common::SystemPaths pluginPath;
-  pluginPath.SetPluginPathEnv(kRenderPluginPathEnv);
+
+  // TODO(CH3): Deprecated. Remove on tock.
+  std::string result;
+  if (!env(kRenderPluginPathEnv, result))
```

Comment:
Compilation is failing:

```
[01m[K/home/jenkins/workspace/ignition_gazebo-ci-pr_any-ubuntu_auto-amd64/ign-gazebo/src/rendering/RenderUtil.cc:2510:8:[m[K [01;31m[Kerror: [m[K‘[01m[Kenv[m[K’ was not declared in this scope; did you mean ‘[01m[Kgz::common::env[m[K’?
```

Reply:
:thinking:

Reply:
https://github.com/gazebosim/gz-sim/pull/1518/commits/3c17b825855880269b660e3e34dcdf1e8062c0e8